public/modules/custom/helfi_kymp_content/helfi_kymp_content.module

```diff
@@ -313,6 +313,7 @@ function helfi_kymp_content_helfi_paragraph_types() : array {
           'district_and_project_search',
           'district_listing',
           'list_of_plans',
+          'ploughing_schedule',
```

Comment:
I believe this should be allowed for `field_content` of `page` since that is how the paragraph [is currently configured](https://github.com/City-of-Helsinki/drupal-helfi-kymp/blob/dev/conf/cmi/field.field.node.page.field_content.yml#L67). This allows the paragraph for `landing_page` content type and the lines 304-305 configure the `page` content type.

Reply:
Changed